samples/packages/pulumi-typescript/src/package-network-config.ts

```diff
@@ -0,0 +1,84 @@
+import * as cdf from "@openfabr/cdf";
+import * as awsx from "@pulumi/awsx";
+
+
+
+export enum NetworkSubnetType {
+  PUBLIC = 'public',
+  PRIVATE = 'private',
+  ISOLATED = 'isolated',
+  DEFAULT = PUBLIC,
+}
```

Comment:
Things like this - would you want to export them via `index.ts` as extra exports so that it helps users of this package to have some defined constants to play with?

Reply:
rather not export you mean?